### DISCUSSION TITLE
How to copy html with style of current theme

### DISCUSSION THREAD

**Discussion by zw-95** [mvdkwast/obsidian-copy-as-html#57] [Q&A]
I tried to put the theme's CSS in the Custom Style sheet, but it didn't work.

**Comment by mvdkwast** [comment]
Hi @zw-95 , unfortunately there are a lot of processing steps to transform the markdown into HTML, and the resulting HTML is not the same as the one you see in Obsidian. So I'm afraid I see no option to implement this. If you're coming up with a nice custom theme, feel free to share !
